samples/sensor/sht3xd/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _sht3xd:
+
+SHT3XD: High accuracy digital I2C humidity sensor
+#################################################
+
+Description
+***********
+
+This sample application periodically (2 Hz) measures the ambient
+temperature and humidity. The result is written to the console.
+Optionally, it also shows how to use the upper threshold triggers.
+
+References
+**********
+
+ - `SHT3X-DIS sensor <https://www.sensirion.com/en/environmental-sensors/humidity-sensors/digital-humidity-sensors-for-various-applications/>`
```

Comment:
Sorry, my bad, there needs to be a trailing underscore:

```suggestion
 - `SHT3X-DIS sensor <https://www.sensirion.com/en/environmental-sensors/humidity-sensors/digital-humidity-sensors-for-various-applications/>`_
```

Reply:
No, my bad; I missed that when transcribing the change (it's years since I used restructured text).  Is it fixed now?